tr.org.pardus.pen.yaml

```diff
@@ -0,0 +1,29 @@
+app-id: tr.org.pardus.pen
+runtime: org.kde.Platform
+runtime-version: 6.7
+sdk: org.kde.Sdk
+command: pardus-pen
+finish-args:
+  - --socket=x11
+  - --device=dri
+modules:
+
+  - name: pardus-pen
+    buildsystem: meson
+    build-options:
+      env:
+        PATH: /app/bin:/usr/libexec:/usr/bin
```

Comment:
```suggestion
```

this is not necessary.

Reply:
rcc command available in /usr/libexec. so required for building.

Reply:
use `append-path` instead of `env`

Reply:
okay. I changed